os-autoinst/os-autoinst-distri-opensuse
# Enable irqbalance test in MU tests

test scenario:

- ensure guests with at least 4 vcpu core and irqbalance installed.
- stop irqbalance.service in guest if it is running.
- bind all NIC IRQs to cpu0 & check if binding is successful by reading the smp affinities(all the values should be '1'(cpu0)) in guest.
- start irqbalance.service again.
- check smp affinities are changed to be distributed among CPUs.
- get the total interrupts on each CPUs.
- ping multiple URLs to generate NIC interrupts on different IRQs.
- get the total interrupts on each CPUs, check if the new increased interrupts are distributed among CPUs

Related ticket: https://progress.opensuse.org/issues/92815
Verification run:
[multiple guests in my own openqa server](http://10.67.129.102/tests/119)
[15sp3 guests in 15sp4 host in OSD](https://openqa.nue.suse.com/tests/8279264)
[MU tests with 15sp3 xen host](http://openqa.qam.suse.cz/tests/37776)
[MU tests with 12sp5 xen host](http://openqa.qam.suse.cz/tests/38110) 
ignore this test: [MU tests with 15sp2 xen host](http://openqa.qam.suse.cz/tests/37862)   15sp2 host install fail due to LTSS code

@alice-suse @waynechen55 @guoxuguang @nanzhg @tbaev @tonyyuan1 @pdostal  Welcome review!
